### PR TITLE
JWT Updates, refactoring

### PR DESCRIPTION
**Pull request by TimGraf**
@gsiegman @AlexMaskovyak @RegoAttainia @JonoMercier @arizonatribe @mgonzo @atomboulian added as reviewers.

- Updated README, I'll expand on this later
- JWT updated with organization
- Support for superuser
- Add RPC server to settings
- Added comments
- Refactored so that the whole decoded JWT is set on the request object once authenticated, permissions class use the decoded JWT from `request.user`
- `scopes` claim renamed to `scope`, to be more standard, neither is a registered claim but `scope` is used by more implementations

### CODE REVIEW

File: attainia_django_extensions/jwt/jwt_scope_permissions.py [hunk 1]
```diff
@@ -46,35 +44,35 @@ class JwtScopePermission(permissions.BasePermission, RpcMixin):
...
         view_class = None
-        scopes = token_response["scopes"]
+        scopes = token_response["scope"]
 
         if inspect.isclass(view):
             view_class = view.__name__
         else:
             view_class = view.__class__.__name__
 
+        if "superuser" in scopes:
```
Comment (AlexMaskovyak) [inline attainia_django_extensions/jwt/jwt_scope_permissions.py]: I'm somewhat concerned about the magic string and that we have the superuser embedded here as a do-anything scope.

Comment (TimGraf) [reply attainia_django_extensions/jwt/jwt_scope_permissions.py]: We can definitely have the superuser scope as a configurable string.  As far as what a super user can and can’t do, I’ll need further direction or perhaps it’s something we want to discuss with the team.

File: attainia_django_extensions/jwt/jwt_scope_permissions.py [hunk 2]
```diff
@@ -46,29 +53,28 @@ class JwtScopePermission(permissions.BasePermission, RpcMixin):
...
         try:
-            token: str = get_authorization_header(request).decode().split()[1]
-            self.logger.debug("Validating token: %s", token)
-
-            token_resp = self.call_service_method("auth_service", "validate_token", False, token)
+            token_resp = request.user
             self.logger.debug("Token Response: %s", token_resp)
 
-            if self._token_includes_scope(token_resp, view, request.method):
+            if token_resp["role"] == settings.USER_ROLES["superuser"]:
```
Comment (AlexMaskovyak) [inline attainia_django_extensions/jwt/jwt_scope_permissions.py]: Will this explode if "role" isn't included the jwt? I'd prefer it if we were mindful of that and didn't cause a 500.

Comment (TimGraf) [reply attainia_django_extensions/jwt/jwt_scope_permissions.py]: Good point I can use `getattr()` but the auth service is adding a role claim to every JWT.

**Comment by gsiegman** [comment]
Is this ok for me to merge? If so, I'll merge here and push to PyPi.

**Comment by TimGraf** [comment]
I’m ok with it.

**Comment by AlexMaskovyak** [comment]
I think I'm good with this for a first cut.

**Pull request merged**